helpers/helpers.go

```diff
@@ -222,18 +222,6 @@ func Download(filename string, url string) (err error) {
 	return nil
 }
 
-// GetDirContents is an an assert-style helper to get the contents of a
-// directory, or to exit on failure.
-func GetDirContents(dirname string) []os.FileInfo {
```

Comment:
Why?

Reply:
This function was not being used anymore.

I've moved the only user of the function to use ioutil.ReadDir directly, because I did not want the "PrintError/os.Exit" which was the only extra thing provided by GetDirContents.

Reply:
Ok :+1:  :)